packages/arb-token-bridge-ui/src/state/app/state.ts

```diff
@@ -53,9 +53,9 @@ export interface MergedTransaction {
   sender?: string
   destination?: string
   direction: TxnType
-  status: string // TODO: Use enums
-  createdAt: string | null
-  resolvedAt: string | null
+  status?: string // TODO: Use enums
```

Comment:
Why is `status` optional here?

Reply:
It's because status isn't mapped to MergedTransaction right away. First we fetch a list of MergedTransactions without status, and then we map statuses starting from the earliest transactions.

There's an alternative where we have separate types for this, so like MergedTransaction without and with status. What we could do is Omit status instead of making it optional. That sounds good?

Reply:
Yes, omitting status from MergedTransactions with naming like `MergedTransactionsBeforeStatusFetch` might be much better at understanding our new logic.

Reply:
We decided that it will bring extra complexity to do Omit here (for now). We will wait until we remove more code in the incoming PRs and see what's the best solution for status